test/run-integration-tests

```diff
@@ -60,12 +61,14 @@ prefix() {
 #
 fetch-ringpop-common() {
     if [ ! -e "$ringpop_common_dir" ]; then
-        run git clone --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir"
+        run git clone --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir" --branch "$ringpop_common_branch"
```

Comment:
Should we instead run the integration test for both languages, and see if we can report an expected failure?

Discussed with @benfleis as well, that strikes us a more reasonable approach.